runtime/tests/interpreter/builtinfunctions_test.go

```diff
@@ -147,6 +149,92 @@ func TestInterpretToBytes(t *testing.T) {
 	})
 }
 
+func TestInterpretAddressFromBytes(t *testing.T) {
+
+	t.Parallel()
+
+	runValidCase := func(t *testing.T, expected []byte, innerCode string) {
+		t.Run(innerCode, func(t *testing.T) {
+			t.Parallel()
+
+			code := fmt.Sprintf(`
+                  fun test(): Address {
+                      return Address.fromBytes(%s)
+                  }
+            	`,
+				innerCode,
+			)
+
+			inter := parseCheckAndInterpret(t, code)
+			res, err := inter.Invoke("test")
+
+			require.NoError(t, err)
+
+			addressVal, ok := res.(interpreter.AddressValue)
+			require.True(t, ok)
+
+			require.Equal(t, expected, addressVal.ToAddress().Bytes())
+		})
+	}
+
+	runValidRoundTripCase := func(t *testing.T, innerCode string) {
+		t.Run(innerCode, func(t *testing.T) {
+			t.Parallel()
+
+			code := fmt.Sprintf(`
+                  fun test(): Bool {
+                    let address : Address = %s;
+					return address == Address.fromBytes(address.toBytes());
+                  }
+            	`,
+				innerCode,
+			)
+
+			inter := parseCheckAndInterpret(t, code)
+			res, err := inter.Invoke("test")
+
+			require.NoError(t, err)
+
+			boolVal, ok := res.(interpreter.BoolValue)
+			require.True(t, ok)
+
+			require.True(t, bool(boolVal))
+		})
+	}
+
+	runInvalidCase := func(t *testing.T, innerCode string) {
+		t.Run(innerCode, func(t *testing.T) {
+			t.Parallel()
+
+			code := fmt.Sprintf(`
+                  fun test(): Address {
+                      return Address.fromBytes(%s)
+                  }
+            	`,
+				innerCode,
+			)
+
+			inter := parseCheckAndInterpret(t, code)
+			_, err := inter.Invoke("test")
+
+			RequireError(t, err)
+			require.ErrorAs(t, err, &common.AddressOverflowError)
```

Comment:
`ErrorAs` writes into the target pointer. Better to use `ErrorIs` for the equality check:

```suggestion
			require.ErrorIs(t, err, common.AddressOverflowError)
```